components/integration/position/Position.module.css

```diff
@@ -0,0 +1,112 @@
+.box div {
+    background-color: #eeeeee;
+}
+
+.box .wrapper {
+    height: 217px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 20px;
+}
+
+/* .box div p {
+    margin-bottom: 0 !important;
+} */
+
+.boxGreen div {
+    background-color: #20b6b0;
+    display: flex;
+    justify-content: space-between;
+}
+
+/* .boxGreen div div {
+    display: inline-block;
+} */
+
+/* .boxGreen div {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+} */
+
+.boxGreen .middle {
+    padding-left: 20px;
+    padding-right: 20px;
+    /* display: block; */
+    display: flex;
+    height: 217px;
+    margin: 0 auto;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.boxGreen .middle img {
+    /* width: -moz-available;         
+  width: -webkit-fill-available;  
+  width: fill-available; */
+    width: 100%;
+    margin-bottom: 20px;
+    display: block;
+}
+
+.boxGreen p {
+    display: block;
+    font-size: 20px;
+    font-weight: 400;
+    text-align: center;
+    color: #ffffff;
+}
+
+.triangleLeft {
+    background-color: #ffffff !important;
+    -webkit-mask: url('/images/play=button.svg') no-repeat center;
+    mask: url('/images/play=button.svg') no-repeat center;
+    /* margin-left: -2px; */
+}
+
+.triangleRight {
+    background-color: #ffffff !important;
+    -webkit-mask: url('/images/play=button.svg') no-repeat center;
+    mask: url('/images/play=button.svg') no-repeat center;
+    -webkit-transform: rotate(180deg);
+    -moz-transform: rotate(180deg);
+    -ms-transform: rotate(180deg);
+    -o-transform: rotate(180deg);
+    transform: rotate(180deg);
+    /* margin-right: -1px; */
+}
+
+
+
+/* .colored {
```

Comment:
is there any reason to keep this comment or can we just remove it?

Reply:
I kept it, in case we want to use the style for hover effect.